assignments/boolean-birds/code.js

```diff
@@ -15,4 +15,18 @@
  *   - Flibble Birds eat fish, mice, and worms.
  *   - Globby Birds eat fish, mice, and nuts.
  *
- */
+ */ 
+const isFlobbyBird = (a,b) => a&&b
```

Comment:
This looks great. You can reformat this code by right clicking in the editor pane and choosing "Format document" from the menu that pops up. It will add some spaces that will make this a bit more legible. Something like this: 

```suggestion
const isFlobbyBird = (a, b) => a && b;
```